File: API/OrderDispatchService/OrderDispatch.Api/Orders/OrderResponse.cs
namespace OrderDispatch.Api.Orders;

public record OrderResponse(
    Guid Id,
    int TableNo,
    string Status,
    string Type,
    decimal Total,
    DateTimeOffset CreatedAt,
    string? DeliveryAddress,
    string? ClientPhone,
    string? ClientName,
    bool IsPaid,
    DateTimeOffset? PaidAt,

    // --- НОВЕ ПОЛЕ ---
    int? WaiterId,

    List<OrderItemResponse> Items
);

public record OrderItemResponse(
    int Id,
    int DishId,
    string DishTitle,
    int Qty,
    decimal Price,
    int StationId,
    string Status
);

public record UpdateItemStatusRequest(Domain.Entities.OrderItemStatus Status);

File: ClientSide/UkrainianStyleRestaurant_2.0_another/app/src/main/java/com/example/ukrainianstylerestaurant/model/OrderResponse.java
package com.example.ukrainianstylerestaurant.model;

import java.util.List;

public class OrderResponse {
    public String id;
    public int tableNo;
    public String status;
    public String type;
    public double total;
    public String createdAt;
    public String deliveryAddress;
    public String clientPhone;
    public String clientName;

    public boolean isPaid;
    public String paidAt;

    public List<OrderItemResponse> items;
}